src/main/frontend/src/components/landingpage/FeatureItem.module.css

```diff
@@ -0,0 +1,50 @@
+.featureitemimg {
```

Comment:
please when className has more than one word split them with `-`or` _` it is lot easier to read